app/src/main/java/fr/free/nrw/commons/contributions/ContributionComparator.java

```diff
@@ -0,0 +1,11 @@
+package fr.free.nrw.commons.contributions;
+
+import java.util.Comparator;
+
+public class ContributionComparator implements Comparator<Contribution> {
+
+  @Override
+  public int compare(Contribution c1, Contribution c2) {
+    return c1.dateUploaded.compareTo(c2.dateUploaded);
```

Comment:
I'm not really sure if this would happen practically but I believe there might chance of `c1.dateUploaded` being null (just as Android Studio suggests). To add to the suspicion, the `dateUploaded` field itself seems to be marked as `@Nullable`. So, I think it might be better to handle the possible `null`. I'll defer the decision to the other reviewers, though.

Reply:
@sivaraam you are right dateUploaded is marked @Nullable hence my decision to use dateCreated. but since the value provided for them is synonymous it will work for now. Let's see what other reviewers have to say like you said... thanks